src/TSMapEditor/UI/CursorActions/DeleteTubeCursorAction.cs

```diff
@@ -25,7 +25,7 @@ public override void DrawPreview(Point2D cellCoords, Point2D cameraTopLeftPoint)
             cellPixelCoords = new Point2D(cellPixelCoords.X + Constants.CellSizeX / 2, cellPixelCoords.Y);
             cellPixelCoords = cellPixelCoords.ScaleBy(CursorActionTarget.Camera.ZoomLevel);
 
-            const string text = "Delete Tunnel";
+            string text = Translate("Text", "Delete Tunnel");
```

Comment:
Might as well fix the inconsistency between the name and the instruction text.

```suggestion
            string text = Translate("Text", "Delete Tube");
```